stravalib/client.py

```diff
@@ -351,67 +318,6 @@ def update_athlete(
         raw_athlete = self.protocol.put("/athlete", **params)
         return model.Athlete.parse_obj(raw_athlete)
 
-    def get_athlete_followers(self, athlete_id=None, limit=None):
-        """
-        Gets followers for current (or specified) athlete.
-
-        :param: athlete_id
-        :type athlete_id: int
-
-        :param limit: Maximum number of athletes to return (default unlimited).
-        :type limit: int
-
-        :return: An iterator of :class:`stravalib.model.Athlete` objects.
-        :rtype: :class:`BatchedResultsIterator`
-        """
-        if athlete_id is None:
-            result_fetcher = functools.partial(
-                self.protocol.get, "/athlete/followers"
-            )
-        else:
-            raise NotImplementedError(
-                "The /athletes/{id}/followers endpoint was removed by Strava.  "
-                "See https://developers.strava.com/docs/january-2018-update/"
-            )
-            # result_fetcher = functools.partial(self.protocol.get,
-            #                                    '/athletes/{id}/followers',
-            #                                    id=athlete_id)
-
-        return BatchedResultsIterator(
-            entity=model.Athlete,
-            bind_client=self,
-            result_fetcher=result_fetcher,
-            limit=limit,
-        )
-
-    def get_both_following(self, athlete_id, limit=None):
-        """
-        Retrieve the athletes who both the authenticated user and the indicated
-         athlete are following.
-
-        This endpoint was removed by Strava in Jan 2018.
-
-        :param athlete_id: The ID of the other athlete (for follower intersection with current athlete)
-        :type athlete_id: int
-
-        :param limit: Maximum number of athletes to return. (default unlimited)
-        :type limit: int
-
-        :return: An iterator of :class:`stravalib.model.Athlete` objects.
-        :rtype: :class:`BatchedResultsIterator`
-        """
-        raise NotImplementedError(
-            "The /athletes/{id}/both-following endpoint was removed by Strava.  "
-            "See https://developers.strava.com/docs/january-2018-update/"
-        )
-        # result_fetcher = functools.partial(self.protocol.get,
-        #                                    '/athletes/{id}/both-following',
-        #                                    id=athlete_id)
-        #
-        # return BatchedResultsIterator(entity=model.Athlete,
-        #                               bind_client=self,
-        #                               result_fetcher=result_fetcher,
-        #                               limit=limit)
 
     # TODO: Can't find this in the api documentation either. Does it still work?
```

Comment:
`/athletes/{id}/koms` is undocumented but still seems to work. We may want to add a user warning here. It may be a good idea to make a separate issue for systematically checking which client methods are still documented, just silently supported, or have stopped working altogether.

Reply:
AHHH ok that sounds good. do you want me to open issues to atleast help a little bit here ? i can atleast do that from this PR so you can keep moving on code. 

i can spend a bit more time working through running the various attributes and such later this week as well but we could still merge and i can open issues if i find other code related issues while testing methods and attrs.

Reply:
See #323 . I'm in favor of letting that wait until this work is finished - I'm a very bad multitasker ¯\_(ツ)_/¯